### PR TITLE
MetricsDownloader Models

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: django_refugeerights/metricsdownloader/models.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+
+class Metric(models.Model):
+    lookup_name = models.CharField(max_length=255, null=False, blank=False)
+    display_name = models.CharField(max_length=25, null=False, blank=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __unicode__(self):
+        return "Metric %s" % (self.display_name)
```
Comment (imsickofmaps) [inline django_refugeerights/metricsdownloader/models.py]: Not unicode

File: django_refugeerights/metricsdownloader/models.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+class MetricRecord(models.Model):
+    metric = models.ForeignKey(Metric, related_name='metric', null=False)
+    date_recorded = models.DateTimeField(null=False, blank=False)
+    value = models.IntegerField(null=False, blank=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __unicode__(self):
+        return "%s as on %s" % (self.metric, self.date_recorded)
```
Comment (imsickofmaps) [inline django_refugeerights/metricsdownloader/models.py]: Not unicode.

File: django_refugeerights/metricsdownloader/models.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    display_name = models.CharField(max_length=25, null=False, blank=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __unicode__(self):
+        return "Metric %s" % (self.display_name)
+
+
+class MetricRecord(models.Model):
+    metric = models.ForeignKey(Metric, related_name='metric', null=False)
```
Comment (imsickofmaps) [inline django_refugeerights/metricsdownloader/models.py]: make `related_name = 'records'` remember it's in reverse

**Comment by imsickofmaps** [comment]
Tests?

**Comment by gsvr** [comment]
As discussed tests not required for models - no Api endpoints

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**